packages/sdk/client/src/services/local-client-services.ts

```diff
@@ -19,26 +20,39 @@ import { trace } from '@dxos/tracing';
 export const fromHost = async (
   config = new Config(),
   params?: ClientServicesHostParams,
+  observabilityGroup?: string,
+  signalTelemetryEnabled?: boolean,
 ): Promise<ClientServicesProvider> => {
-  return new LocalClientServices({
+  const getSignalMetadata = () => {
+    return {
+      ...lcs.signalMetadataTags,
+      ...(observabilityGroup ? { group: observabilityGroup } : {}),
+    };
+  };
+  const lcs = new LocalClientServices({
```

Comment:
```suggestion
  const services = new LocalClientServices({
```
uncommon abbreviations invite too many questions 😅